### DISCUSSION TITLE
caption: and bounding box overflow

### DISCUSSION THREAD

**Discussion by olsln** [ImageMagick/ImageMagick#1926] [General] [answered]
I read http://www.imagemagick.org/Usage/text/#overflow and experimented a bit. However, I don't get how these two images render so differently:

1. `$ convert -size 300x300 -background lightblue -font Loki-Cola caption:"Loki C" 1.jpg`
![1](https://user-images.githubusercontent.com/64224741/80139527-e4eca200-85a6-11ea-8ce7-5c3d080edc85.jpg)
2. `$ convert -size 300x200 -background lightblue -font Loki-Cola caption:"Loki C" 2.jpg`
![2](https://user-images.githubusercontent.com/64224741/80139563-f03fcd80-85a6-11ea-864c-93dd2009c394.jpg)

The first one gives me a font size where the C will be cut off at its character overflow.
The second one gives me a font size such that the C fully fits in the image width.

Is it supposed to be that different although the image only differs in height?

Version: ImageMagick 6.8.9-9 Q16 x86_64 2019-11-12 http://www.imagemagick.org

**Comment by snibgo** [comment, marked as answer]
I suggest you try with a current version of IM.

**Comment by olsln** [comment]
Thanks, it's fixed in the current version.
